app/celer-sim/simple-driver.py

```diff
@@ -146,8 +146,9 @@ def strtobool(text):
 time = run_output['time'].copy()
 steps = time.pop('steps')
 if use_device:
-    assert(len(steps) == run_output['num_step_iterations'][0])
+    assert len(steps) == 1
```

Comment:
I think it's 3 😉

Reply:
But I thought with "merge events" it would only have 1? The dimension being number of streams...

Reply:
Isn't `merge_events` false in the test?